hypothesis-python/setup.py

```diff
@@ -96,7 +96,7 @@ def local_file(name):
     zip_safe=False,
     extras_require=extras,
     install_requires=[
-        "attrs>=19.2.0",
+        "attrs>=22.2.0",
```

Comment:
I was pretty nervous about this, but [checking download stats](https://www.pepy.tech/projects/attrs?versions=23.*&versions=22.*&versions=21.*&versions=20.*&versions=19.*) it seems likely to be fine 🙂